Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -58,27 +55,9 @@
 
 upgradeProperties[2017] = {
     '2017' : {
-        'Geom' : 'DB:Extended',
-        'GT' : 'auto:phase1_2017_realistic',
-        'Era' : 'Run2_2017',
-        'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','HARVESTFull'],
-    },
-    '2017NewFPix' : {
-        'Geom' : 'Extended2017NewFPix',
-        'GT' : 'auto:phase1_2017_realistic',
-        'Era' : 'Run2_2017_NewFPix',
-        'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','HARVESTFull'],
-    },
-    '2017HCALdev' : {
-        'Geom' : 'Extended2017dev',
-        'GT' : 'auto:phase1_2017_hcaldev',
-        'Era' : 'Run2_2017_HCALdev',
-        'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','HARVESTFull'],
-    },
-    '2017AllNew' : {
         'Geom' : 'Extended2017new',
         'GT' : 'auto:phase1_2017_hcaldev',
```

Comment:
@makortel, please go ahead and swap the names in autoCond within this PR and remove completely the `auto:phase1_2017_hcaldev` (will sign the changes in this PR)
Note so selves (AlCa): when the HCal geometry issue is sorted out we need 
   * to restore the correct label in the geometry (not Extended2017dev, but DB:Extended) 
   * to pull in the corresponding Tracker Alignment changes.